bgpd/bgp_bmp.c

```diff
@@ -314,11 +369,26 @@ static void bmp_put_info_tlv(struct stream *s, uint16_t type,
 	stream_put(s, string, len);
 }
 
+static void __attribute__((unused))
+bmp_put_vrftablename_info_tlv(struct stream *s, struct bmp *bmp)
```

Comment:
Where is this used?

Reply:
it's not yet but will be in upcoming bmp features, hence the function is annotated with `__attribute__((unused))` in the highlighted line.